### ISSUE TITLE
Flinging only works until performing an overscroll on the same side 

### ISSUE THREAD

**Issue by Emplexx** [Valkriaine/bouncy#19]
When I fling my list and it reaches the end, it bounces as expected, but if I drag on the same side (perform an overscroll) and then fling again, it no longer bounces and just abruptly stops. Eg if I drag (overscroll) on the bottom side of the list, flinging animation will no longer work on the bottom

The `BouncyRecyclerView` is in a `CoordinatorLayout`. The layout file is for a fragment. My adapter extends `BouncyRecyclerView`. 
`BouncyRecyclerView` has the following settings: 
```
app:recyclerview_fling_animation_size="0.1"
app:recyclerview_overscroll_animation_size="0.5"
app:recyclerview_damping_ratio="DAMPING_RATIO_LOW_BOUNCY"
app:recyclerview_stiffness="STIFFNESS_LOW"```

**Comment by valkriaine** [comment]
> When I fling my list and it reaches the end, it bounces as expected, but if I drag on the same side (perform an overscroll) and then fling again, it no longer bounces and just abruptly stops. Eg if I drag (overscroll) on the bottom side of the list, flinging animation will no longer work on the bottom
> 
> The `BouncyRecyclerView` is in a `CoordinatorLayout`. The layout file is for a fragment. My adapter extends `BouncyRecyclerView`. 
> `BouncyRecyclerView` has the following settings: 
> ```
> app:recyclerview_fling_animation_size="0.1"
> app:recyclerview_overscroll_animation_size="0.5"
> app:recyclerview_damping_ratio="DAMPING_RATIO_LOW_BOUNCY"
> app:recyclerview_stiffness="STIFFNESS_LOW"```

Hello, are you using a Samsung galaxy device?

**Comment by Emplexx** [comment]
Nope, I am using a Google Pixel 3, running the last update it received from October 2021.
